conans/test/integration/toolchains/meson/test_mesontoolchain.py

```diff
@@ -517,3 +518,19 @@ def generate(self):
     client.save({"conanfile.py": conanfile})
     client.run("install .", assert_error=True)
     assert "You can only pass native=True if you're cross-building" in client.out
+
+
+def test_compiler_path_with_spaces():
+    conanfile = (GenConanfile().with_name("lib").with_version("1.0").with_generator("MesonToolchain")
+                 .with_settings("compiler"))
+    profile = textwrap.dedent("""
+        include(default)
```

Comment:
This is an integration test so it does not matter if you run this in macOS, Linux, or wherever. You could have kept the previous one without requiring Windows as you're not compiling anything:

```ini
        [settings]
        arch=x86
        os=Windows
        compiler=msvc
        compiler.version=193
        compiler.runtime=static
        build_type=Release
```

Reply:
Good to know!